### ISSUE TITLE
Parameters/HyperParameters for training HICO dataset

### ISSUE THREAD

**Issue by zijunwei** [rohitgirdhar/AttentionalPoolingAction#19]
Hi I'm trying to reproduce the the performance on HICO dataset. Using ResNet only you reported the performance of 30.2 mAP. But I only got 26.3. Could you please share the parameters you use?
These parameters are:

1. How the image is pre-processed for training/validating
2. Are there any other information used (pose?text?)
3. Is the loss function softmax cross entropy or sigmod binary cross entropy for multi-class?
4. what's the optimizer? (adam or SGD?)
5. what's the training scheduler? (learning rate change, freezing any layers?)

Thanks a lot!

**Comment by rohitgirdhar** [comment]
1. Same as what I do for MPII, random crops, flips etc. Input image size is 480px
2. No
3. Softmax cross-entropy. I randomly pick a class the image belongs to each time it is seen at train time. 
4. Trained over 4 GPUs using sync-SGD, I use LR=0.001, batch size=16 per GPU, LR decay by 0.1 every 5000 iterations, ad I train a total of 12000 iterations.

**Comment by zijunwei** [comment]
Thanks, closing this issue
